source/ui/x11/X11Window.ooc

```diff
@@ -52,8 +64,34 @@ X11Window: class extends NativeWindow {
 	}
 	resize: func (size: IntSize2D) { XResizeWindow(this display, this backend, size width, size height) }
 	free: override func {
+		if (this _xImage)
+			XDestroyImage(this _xImage)
+		XFreeGC(this display, this _x11GraphicsContext)
+		XDestroyWindow(this display, this backend)
 		XCloseDisplay(this display)
 		super()
 	}
+	draw: func (image: RasterYuv420Semiplanar) {
```

Comment:
I'm asking because now `operator[]` is just declared in some image classes, without any base class interface for it, so now it's not possible to accept a *generic* image type and use `operator[]` on it.